### ISSUE TITLE
Allow users to suggest Alt Text or Moderate Alt Text

### ISSUE THREAD

**Issue by nuke** [mastodon/mastodon#19907]
### Pitch

Allow users to suggest Alt Text and have Alt Text Moderator roles for trusted people to update alt text.
Suggestions can be approved by owners of posts containing images.
Moderation can be done by special-role server admins.

### Motivation

Part of the issue with alt text is that many users are simply not able to understand or process how to add appropriate alt text to an image. This becomes an issue of access not just for the blind user, but for users with other social issues such as autism or ADHD, as they may not understand how to communicate the important information or have the attention span to do so. It would be useful to allow users to suggest alt text or alt text adjustments for an image if they are more comfortable with creating Alt Text. Most likely, it would be good to have users who own the post with the image to approve suggestions given by other users.
However, it may also be prudent to have moderators specific to alt text who can update or remove alt text from posts if it's deemed inappropriate.

**Comment by mgifford** [comment]
Interesting to think of delegating this responsibility to another person. Interesting to think about a community where this would even be proposed. Quite cool.

**Comment by orowith2os** [comment]
How would this work across instances? Would there be an alt text change suggestion on one instance that's sent to another to be verified by the moderators on the instance getting the suggestion?

**Comment by hkolbeck** [comment]
I thought it was worth throwing this in here, but I'm working on a related project for the entire internet, https://alt-text.org

**Comment by rkingett** [comment]
How would this work with Authorized fetch, too?

**Comment by ryan-augustinsky** [comment]
It would function if this were moderated at the local-level, where the addition of alt-text works like an update to the post, and federates without any extra code.

I envision an interface in the moderation panel that lists posts without alt text. 

When a moderator adds alt text, the status update creator is notified. 

There could be an opt-out option for users who don't want moderators describing their selfiies, etc.

**Comment by ThisIsMissEm** [comment]
This would need a [FEP written](https://codeberg.org/fediverse/fep/src/branch/main/README.md#submitting-a-fep) as to be able to suggest alt text for statuses from remote instance.

I think there's perhaps three features here:
1. Allow moderators to create "Alt Text Proposals"
2. Allow users to propose alt text for local statuses
3. Allow users to propose alt text for remote statuses.

Once a proposed alt text is approved, it'd be handled the same way as a status update or edit.

It may be worth splitting this issue into those three at some point.

cc @stefanbohacek

**Comment by OddDev** [comment]
Duplicate of https://github.com/mastodon/mastodon/issues/26464 I guess
